Objects/listobject.c

```diff
@@ -272,6 +272,15 @@ PyList_GetItemRef(PyObject *op, Py_ssize_t i)
     return Py_NewRef(PyList_GET_ITEM(op, i));
 }
 
+static inline PyObject*
+_PyList_GetItemRef(PyListObject *op, Py_ssize_t i)
```

Comment:
I think the naming convention is to use the `Py` prefix for global functions (not `static` functions) [^1].

So maybe: `list_get_item_ref` or `list_try_get` or something along those lines.

[^1]: https://peps.python.org/pep-0007/#:~:text=Use%20a%20Py%20prefix%20for%20public%20functions%3B%20never%20for%20static%20functions

Reply:
Done!